packages/astro/client.d.ts

```diff
@@ -1,6 +1,7 @@
 /// <reference types="vite/types/import-meta.d.ts" />
 /// <reference path="./types/content.d.ts" />
 /// <reference path="./types/actions.d.ts" />
+/// <reference path="./jsx-runtime.d.ts" />
```

Comment:
It might just be my development machine. I was having some issues with the devcontainers on windows. 

It wasn't showing the `astroHTML` as a valid option until I added that. Doesn't seem to be an issue anymore. I'll remove now, thank you!